tests/serialization/test_run_configs.py

```diff
@@ -62,9 +63,14 @@ def test_serialize_kubernetes_run(config):
         "memory_request",
         "service_account_name",
         "image_pull_secrets",
+        "always_pull_new_image",
     ]
     for field in fields:
-        assert getattr(config, field) == getattr(config2, field)
```

Comment:
I think the serialization field type is causing issues here, we should not have to change the test logic here